sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Infrastructure/BufferedPublisher.cs

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Messaging.EventHubs.Tests;
+
+namespace Azure.Messaging.EventHubs.Stress
+{
+    internal class BufferedPublisher
+    {
+        private string _connectionString;
+        private string _eventHubName;
+        private Metrics _metrics;
+        private BufferedProducerTestConfig _testConfiguration;
+
+        public BufferedPublisher(BufferedProducerTestConfig testConfiguration,
+                                 Metrics metrics)
+        {
+            _connectionString = testConfiguration.EventHubsConnectionString;
+            _eventHubName = testConfiguration.EventHub;
+            _metrics = metrics;
+            _testConfiguration = testConfiguration;
+        }
+
+        public async Task Start(CancellationToken cancellationToken)
+        {
+            var enqueueTasks = new List<Task>();
+
+            while (!cancellationToken.IsCancellationRequested)
+            {
+                using var backgroundCancellationSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
+
+                // Create the buffered producer client and register the success and failure handlers
+
+                var options = new EventHubBufferedProducerClientOptions
```

Comment:
The Buffered Producer has some adjustments to the standard defaults for retry policy if the options aren't explicitly passed.    We should mirror them for the test, since that'll be our majority case.  

Source:  https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubBufferedProducerClient.cs#L68

Reply:
If you have ideas on how we can better detect and adjust retry configuration when the individual values aren't set, I'd be very interested in hearing them.   The best that I can think of is creating a mirrored type inside the buffered options.      Maybe we should explore that as a quick interlude between phase 1 and phase 2.